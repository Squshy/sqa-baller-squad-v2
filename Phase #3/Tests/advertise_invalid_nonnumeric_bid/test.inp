LOGIN
MR.RUNESCAPEMAN
buyginggf20k
ADVERTISE
DRAGON SCIMITAR
AB.99