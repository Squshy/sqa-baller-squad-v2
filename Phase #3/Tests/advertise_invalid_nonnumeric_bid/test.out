=========================================================
                        ADVERTISE
=========================================================

Item name: DRAGON SCIMITAR

Minimum bid: AB.99

---------------------------------------------------------
            A bid must be numeric and can only
         be between $0.01 and $999.99 inclusively.
=========================================================